lib/util/auth.js

```diff
@@ -42,9 +42,9 @@ export function getAuth0Config (persistence) {
  * Gets the callback methods for Auth0.
  * Note: Methods inside are originally copied from https://github.com/sandrinodimattia/use-auth0-hooks#readme
  * and some methods from that example may still be untouched.
- * @param reduxStore The redux store to carry out the navigation action.
+ * @param dispatch The dispatch method to carry out the actions.
  */
-export function getAuth0Callbacks (reduxStore) {
+export function getAuth0Callbacks (dispatch) {
```

Comment:
These callbacks are effectively actions, right? I feel like there are some gymnastics happening to try to make everything a util function instead of an action. I don't really like the idea of passing `dispatch` from the component as a param here. I think this is a spot where things could be simplified by making these actions and connecting them in `mapDispatchToProps`. Conventions exist for a reason: we do things in a common predictable way so that the app can be navigated easily by the rest of the team. Also, I don't think we need to combine them all into a `getAuth0Callbacks` function.

Reply:
Sample code legacy...